src/app/shared/store/filter.state.ts

```diff
@@ -147,10 +147,12 @@ export class FilterState {
   constructor(private appWorkshopsService: AppWorkshopsService) {}
 
   @Action(SetCity)
-  setCity({ patchState, dispatch, getState }: StateContext<FilterStateModel>, { payload }: SetCity): void {
+  setCity({ patchState, dispatch, getState }: StateContext<FilterStateModel>, { payload, isNeedWriteInStorage }: SetCity): void {
```

Comment:
isNeedWriteInStorage  - insufficient naming, we do not need to save the value to the storage, we are doing it to confirm the city. So this value should be renamed to isConfirmedCity - if yes, then we save it to the storage